src/uwtools/cli/run_forecast.py

```diff
@@ -18,12 +19,9 @@ def main() -> None:
     Parses arguments provided by the user and passes to the Forecast driver class to be run.
     """
     args = parse_args(sys.argv[1:])
-    name = "run-forecast"
-    cli_helpers.setup_logging(
-        log_file=args.log_file, log_name=name, quiet=args.quiet, verbose=args.verbose
-    )
+    setup_logging(quiet=args.quiet, verbose=args.verbose)
     forecast_class = getattr(forecast, "%sForecast" % args.forecast_model)
-    experiment = forecast_class(args.config_file, args.machine, log_name=name)
```

Comment:
Here and in much of what follows the diffs involve simplification and removal of unneeded code.